package.json

```diff
@@ -106,8 +107,13 @@
     "ts-jest": "^27.1.4",
     "typescript": "^4.6.3"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@changesets/cli": "^2.26.1"
```

Comment:
does this need to be a regular dep or can it be a dev-dep?

Reply:
Good call - will move to `devDependencies` 👍